cpp/src/aggregation/aggregation.cpp

```diff
@@ -311,7 +333,7 @@ std::vector<std::unique_ptr<aggregation>> aggregation::get_simple_aggregations(
 }
 
 /// Factory to create a SUM aggregation
-template <typename Base = aggregation>
+template <typename Base>
```

Comment:
Was the default not needed?

Reply:
The default has been declared in the `cudf/aggregation.hpp`. The line above is repeated declaration in a cpp file. Keeping such default declarations will trigger a lot of warnings during compilation.

Reply:
Aha, I thought we had warnings as errors. Curious how that made it through CI.